roles/matrix-jitsi/defaults/main.yml

```diff
@@ -41,16 +44,17 @@ matrix_jitsi_xmpp_guest_domain: guest.meet.jitsi
 matrix_jitsi_xmpp_muc_domain: muc.meet.jitsi
 matrix_jitsi_xmpp_internal_muc_domain: internal-muc.meet.jitsi
 matrix_jitsi_xmpp_modules: ''
+matrix_jitsi_xmpp_muc_modules: ''
+matrix_jitsi_xmpp_muc_modules_internal: ''
```

Comment:
I believe those should remain, so that other playbook users can easily see that those are configurable options, similar to "matrix_jitsi_xmpp_modules".

It doesn't hurt to keep blank, but allows users to add new muc modules for Jitsi features. I did confirm that changes to these variables are reflected in the final configuration properly

Reply:
I didn't see `matrix_jitsi_xmpp_muc_modules` or `matrix_jitsi_xmpp_muc_modules_internal` being used in this PR. Or am I missing something?

Reply:
You're right, I should've submitted this as a separate PR to "expose Jitsi Module Configuration". I'll remove it here and submit it separately later.